galaxy/main/models/collection.py

```diff
@@ -68,6 +69,13 @@ def latest_version(self):
             collection=self
         ).latest('pk')
 
+    @property
+    def highest_version(self):
+        versions = CollectionVersion.objects.filter(collection=self)
+        pairs = [(v, semver.Version(v.version)) for v in versions]
+        pairs.sort(key=lambda x: x[1], reverse=True)
+        return pairs[0][0]
```

Comment:
In theory collection can have zero versions.

Reply:
@cutwater I also updated `latest_version` to account for hidden versions and zero versions